zuul.d/nodeset.yaml

```diff
@@ -67,6 +67,8 @@
       - name: ocps
         nodes:
           - crc
+      - name: rh-subscription
+        nodes: [standalone]
```

Comment:
Just a nit but it may be nicer to not use single line list as multiline is standard in the nodesets.yaml files.
```
- name: rh-subscription
  nodes:
    - standalone
```

Reply:
ok